docs/hw/electrical.md

```diff
@@ -17,41 +18,41 @@ The Adapter Board is also visible through the top cover of the robot, where its
 
 ## Power Connections
 ### Unregulated Battery Port
-The **unregulated battery port** is a JST XH-style connector, with pin 1 as the positive terminal of the battery, and pin 2 is the negative terminal.
+The **unregulated battery port** is a JST XH-style connector, with pin 1 as the positive terminal of the battery, and pin 2 as the negative terminal.
 These terminals are labeled on the bottom side of the board and are visible with the robot on its back and the cargo bay removed.
-The board is capable of supplying a maximum of 2 A of current at the current battery voltage, enforced by a PTC resettable fuse in the robot.
+The board is capable of supplying a **maximum of 2 A** of current at the current battery voltage (14.4 V nominal, 12 V min, 16.8 V max), enforced by a PTC resettable fuse in the robot.
 This port is always on when the battery is powered on.
-Be careful not to back-feed the robot through this port.
-
-Do not attempt to use the robot without its battery installed.
+!!! attention "Notice"
+    Do not to back-feed power into the robot through this port, and do not attempt to use the robot without its battery installed.
 
 ### USB-C® Connector
 The **USB-C® connector** provides a USB 2.0 Host connection into the robot with 5.13 V at 3.0 A provided to power downstream connections.
 The power is disabled on this port unless a proper USB-C® downstream device is connected.
 The 5 V source powering this port will shut off when the battery voltage falls below approximately 11.8 V.
 The USB data connection is made only when the USB/BLE toggle switch plunger is slid toward the USB icon.
 
-### Total Power Output
```

Comment:
I disagree with this becoming an info box. The information is extremely important and should be under a subheading. (By doing this, you have put it under the table of contents under the USB C connector, which doesn't make sense.) I have found with Create 2 that people are generally confused about what volts, amps, and watts are, and how they are used. At the very least, I would suggest the title and first sentence be formatted as heading and text; the rest can be in an info box.

Reply:
My thought was that a colored box would be more visible, but I agree with your point about the page structure and navigation. I changed it back to a section but kept the example as a call-out.

Reply:
Looks great; thank you!